meta.yaml

```
@@ -24,7 +24,7 @@ requirements:
   run:
     - python
     - greenlet ==1.1.3
-    - pyee ==8.1.0
+    - pyee >=9.0.0
```

```suggestion
    - pyee ==9.0.4
```

let's keep that? In the past they broke us with a minor version bump.